spec/abstract-operations.html

```diff
@@ -769,16 +769,15 @@ <h1>
     <emu-clause id="sec-samevaluenongeneric" type="abstract operation" oldids="sec-samevaluenonnumeric sec-samevaluenonnumber">
       <h1>
         <del>SameValueNonNumeric</del><ins>SameValueNonGeneric</ins> (
-          _x_: an ECMAScript language value, but not a Number or a BigInt,
-          _y_: an ECMAScript language value, but not a Number or a BigInt,
+          _x_: an ECMAScript language value, but not a Number, a BigInt, a Record, a Tuple, or a Box,
+          _y_: an ECMAScript language value, but not a Number, a BigInt, a Record, a Tuple, or a Box,
```

Comment:
The name of this AO is “non numeric” - it’s intended to handle everything that’s not a numeric type. Why are R/T/B not included here?

Reply:
@ljharb It's `SameValueNonGeneric` in this proposal (which probably doesn't have much meaning?). This function is meant to handle things for which `SameValue` and `SameValueZero` return the same result.

Reply:
ah k, i didn’t notice the renaming. it needs a better name, but i guess the semantic makes sense.